source/braille.py

```diff
@@ -808,6 +808,14 @@ def getFormatFieldBraille(field, isAtStart, formatConfig):
 				# Translators: Displayed in braille for a heading with a level.
 				# %s is replaced with the level.
 				textList.append(_("h%s")%headingLevel)
+	if formatConfig["reportLinks"]:
+		link=field.get("link")
+		oldLink=fieldCache.get("link") if fieldCache is not None else None

```

Comment:
Would it be bad coding style to provide an empty dictionary as default for fieldCache? That would allow you to get rid of the "if fieldCache is not None else None" part